lib/membrane/rtp/session_bin.ex

```diff
@@ -588,4 +600,36 @@ defmodule Membrane.RTP.SessionBin do
     pt || PayloadFormat.get(encoding).payload_type ||
       raise "Cannot find default RTP payload type for encoding #{encoding}"
   end
+
+  defp maybe_handle_twcc(rtp_extensions, pad_ssrc, ctx, state) do
+    # workaround: as TWCC is a transport-wide extension, there should exist only one TWCC child
+    # that handles packets from all incoming streams that have declared support for it
+    {maybe_twcc, rtp_extensions} = Keyword.pop(rtp_extensions, :twcc)
+
+    should_link? = maybe_twcc != nil
+    should_create_child? = not Map.has_key?(ctx.children, :twcc)
+
+    {maybe_twcc_ssrc, state} =
+      if should_link? and should_create_child? do
+        add_ssrc(nil, state)
+      else
+        {nil, state}
+      end
+
+    maybe_link_twcc =
+      if should_link? do
+        &(&1
+          |> via_in(Pad.ref(:input, pad_ssrc))
+          |> then(fn link ->
+            if should_create_child?,
```

Comment:
Normal `do/end` would be better than the shorter versions with `:` :P

Reply:
It's arguable, but will change 😛